setup.py

```diff
@@ -2,7 +2,7 @@
 # vim: set sw=4 et:
 from setuptools import setup, find_packages
 
-__version__ = "0.4.9"
+__version__ = "0.4.10"
```

Comment:
should we make this 0.5.0?

Reply:
Hm, yeah, not a bad idea, will push a commit to update it